receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/utils.go

```diff
@@ -24,9 +24,9 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-// GenerateDummyMetrics generates two dummy metrics
+// generateDummyMetrics generates two dummy metrics
 // TODO: Remove once this is replaced with actuall metrics generation code
-func GenerateDummyMetrics() *agentmetricspb.ExportMetricsServiceRequest {
+func generateDummyMetrics() *agentmetricspb.ExportMetricsServiceRequest {
```

Comment:
Why is this not in a "_test" file?

Reply:
It is tested in the `utils_test.go` file, and I changed it there too

Reply:
Hi @JohnWu20 , I think the question is why do we have the dummy data generation code in a non-testing file? Can we remove it or move it into unit test file?

Reply:
Thanks, removed the function because it is useless.